static/js/compose.js

```diff
@@ -564,8 +564,8 @@ exports.validate = function () {
     return validate_stream_message();
 };
 
-exports.handle_keydown = function (event) {
-    var textarea = $("#compose-textarea");
+exports.handle_keydown = function (event, textarea_id) {
+    var textarea = $("#" + textarea_id);
```

Comment:
Passing in `textarea_id` seems a bit overly constrained to me, although  I understand the rationale, and it wouldn't be hard to extend.  I'm curious what Tim thinks here.

Reply:
We can pass `$(this)` as `textarea` instead, though.

Reply:
I try to avoid relying on `this` when I can, but I think just passing in a var called `textarea` would be fine and have the caller say something like `var textarea = $('#compose-textarea')` instead of `handle_keydown` hard coding it.